gatsby-node.js

```diff
@@ -272,29 +376,31 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
   })
 }
 
+// Only ran when creating component pages
 // Add additional context to translated pages
 // https://www.gatsbyjs.com/docs/creating-and-modifying-pages/#pass-context-to-pages
-exports.onCreatePage = ({ page, actions }) => {
+exports.onCreatePage = async ({ page, actions }) => {
   const { createPage, deletePage } = actions
 
   const isTranslated = page.context.language !== defaultLanguage
   const hasNoContext = page.context.isOutdated === undefined
   const langVersion = getLangContentVersion(page.context.language)
```

Comment:
We're going to need to re-think how we do this, since version is no longer on a language level.